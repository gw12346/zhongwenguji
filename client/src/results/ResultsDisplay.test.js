import React from 'react';
import Renderer from 'react-test-renderer';
import ResultsDisplay from './ResultsDisplay';

describe('ResultsDisplay', () => {
  it('renders', () => {
    const component = Renderer.create(
      <ResultsDisplay
        estimate={1500}
        uncertainty={200}
      />
    );
    expect(component.toJSON()).toMatchSnapshot();
  });
});
