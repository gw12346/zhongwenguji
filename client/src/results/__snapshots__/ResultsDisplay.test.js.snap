// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResultsDisplay renders 1`] = `
<div
  className="container"
>
  <div
    className="youKnow"
  >
    results.youKnow
  </div>
  <div
    className="estimate"
  >
    <span
      className="mean"
    >
      1500
    </span>
    <span
      className="plusMinus"
    >
      Â±
    </span>
    <span
      className="std"
    >
      200
    </span>
  </div>
  <div
    className="measure"
  >
    results.measure
  </div>
</div>
`;
